graphbolt/include/graphbolt/sampled_subgraph.h

```diff
@@ -0,0 +1,69 @@
+/**
+ *  Copyright (c) 2023 by Contributors
+ * @file graphbolt/sampled_subgraph.h
+ * @brief Header file of sampled sub graph.
+ */
+
+#include <torch/custom_class.h>
+#include <torch/torch.h>
+
+namespace graphbolt {
+namespace sampling {
+
+struct SampledSubgraph : torch::CustomClassHolder {
```

Comment:
Let's use this as the principle to decide which info should be carried in this structure.

Which is all the info that required for the following steps for this subgraph should be added in this structure.

There are 2 status of this structure:
- row compacted but not column.
- both row and column are compacted.

Reply:
Added 2 more tensors indicates node compaction info.